package.json

```diff
@@ -1,75 +1,78 @@
-{ 
+{
   "name": "Open-Energy-Dashboard",
   "version": "0.0.0",
   "private": true,
   "license": "MPL-2.0",
   "scripts": {
-    "start": "node ./src/bin/www",
-    "dev": "webpack -d --watch --color --progress",
-    "build": "cross-env NODE_ENV=production webpack -p",
-    "checkHeader": "./src/scripts/checkHeader.sh",
-    "lint": "./node_modules/.bin/eslint src/client/app/**/*.js src/client/app/**/*.jsx src/server/**/*.js src/server/**/*.jsx src/scripts/**/*.js",
-    "test": "./node_modules/.bin/mocha \"src/server/test/**/*.js\"",
-    "createdb": "node ./src/scripts/createDB.js",
-    "addMamacMeters": "node ./src/scripts/addMamacMeters.js",
-    "updateMamacMeters": "node ./src/scripts/updateMamacMeters.js",
-    "createUser": "node ./src/scripts/createUser.js"
+	"start": "node ./src/bin/www",
+	"dev": "webpack -d --watch --color --progress",
+	"build": "cross-env NODE_ENV=production webpack -p",
+	"checkHeader": "./src/scripts/checkHeader.sh",
+	"lint": "./node_modules/.bin/eslint src/client/app/**/*.js src/client/app/**/*.jsx src/server/**/*.js src/server/**/*.jsx src/scripts/**/*.js",
+	"test": "./node_modules/.bin/mocha \"src/server/test/**/*.js\"",
+	"createdb": "node ./src/scripts/createDB.js",
+	"addMamacMeters": "node ./src/scripts/addMamacMeters.js",
+	"updateMamacMeters": "node ./src/scripts/updateMamacMeters.js",
+	"createUser": "node ./src/scripts/createUser.js"
   },
   "dependencies": {
-    "axios": "~0.16.2",
```

Comment:
You should not change all these. Remove all of them except the dependencies you added, and make sure they use a ~ instead of ^